widget/embedded/src/components/TokenList/TokenList.tsx

```diff
@@ -237,16 +237,14 @@ export function TokenList(props: PropTypes) {
                   ) : undefined
                 }
                 description={
-                  !!blockchain?.info &&
-                  !!address &&
-                  blockchain.type !== 'COSMOS'
+                  !!address && blockchain.type !== 'COSMOS'
```

Comment:
Seems you've removed `!!blockchain?.info` beacuse server doesn't return `.info` for TON blockcain in meta.
Please ask them to add it since we need `addressUrl` anyways to open and redirect user to block explorer and showing that token.

Reply:
You can revert your changes I guess

Reply:
I will revert these changes once the backend team updates the TON metadata with the necessary information.

Reply:
After resolving the issue on the server, I reverted my changes.